src/helper.js

```diff
@@ -101,6 +101,9 @@ function ip2hex(address) {
 }
 
 function expandHome(shortenedPath) {
+	if (shortenedPath === "" || shortenedPath === undefined) {
```

Comment:
I would probably change this to just "if(!shortenedPath)" but I don't massively mind.